kafka/consumer/group.py

```diff
@@ -201,6 +204,8 @@ class KafkaConsumer(six.Iterator):
     }
 
     def __init__(self, *topics, **configs):
+        client = configs.pop("client", None)
```

Comment:
The pattern we've been using so far is to put this in the class-level DEFAULT_CONFIG dict. Just add an entry for `'client': None`